Jenkinsfile

```diff
@@ -81,5 +81,50 @@ pipeline {
             // Clean
             sh 'git clean -fdx -e "*.hprof" || echo "git clean failed, exit code $?"'
         }
+        changed {
```

Comment:
A line break before this one would be nice.

Reply:
ok